.eslintrc.json

```diff
@@ -7,11 +7,9 @@
     "mocha": true
   },
   "extends": [
-    "eslint:recommended",
-    "plugin:@typescript-eslint/eslint-recommended",
     "plugin:@typescript-eslint/recommended",
-    "plugin:@typescript-eslint/recommended-requiring-type-checking",
-    "airbnb-base"
+    "prettier/@typescript-eslint",
+    "plugin:prettier/recommended"
   ],
```

Comment:
Deleted lines that have duplicated effects and added prettier related extensions.